components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -63,6 +63,9 @@
     },
 }
 
+# Pseudo component names
+ARCHIVE_MANAGER_PSEUDO_COMPONENT_NAME = "archive_manager"
```

Comment:
We currently have components and targets. I feel like "pseudo component" isn't a great descriptor for this. Junhao had called these one-time jobs in his k8s design doc. I think that could get confusing since we have compression and query jobs.

So how about calling them actions? I.e., `ARCHIVE_MANAGER_ACTION_NAME`? Another alternative is `ACTION_MANAGER_CLI_CMD_NAME`?

Reply:
Used ARCHIVE_MANAGER_ACTION_NAME